packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -2,72 +2,171 @@
 id: Table
 cssPrefix: pf-c-table
 section: components
-propComponents: ['Table', 'TableHeader', 'TableBody', 'EditableSelectInputCell', 'EditableTextCell', 'RowErrors', 'IHeaderRow', 'IRowData', 'IColumn', 'IExtraRowData', 'IExtraColumnData', 'ISortBy', 'IAction', 'ISeparator', 'ICell', 'IRowCell', 'IValidatorDef', 'IRow']
+propComponents:
+  [
+    'Table',
+    'TableHeader',
+    'TableBody',
+    'EditableSelectInputCell',
+    'EditableTextCell',
+    'RowErrors',
+    'IHeaderRow',
+    'IRowData',
+    'IColumn',
+    'IExtraRowData',
+    'IExtraColumnData',
+    'ISortBy',
+    'IAction',
+    'ISeparator',
+    'ICell',
+    'IRowCell',
+    'IValidatorDef',
+    'IRow',
+    'TableComposable',
+    'Thead',
+    'Tbody',
+    'Tr',
+    'Th',
+    'Td',
+    'Caption',
+  ]
 ouia: true
 ---
 
 Note: Table lives in its own package at [@patternfly/react-table](https://www.npmjs.com/package/@patternfly/react-table)!
 
-import {
-  Table,
-  TableHeader,
-  TableBody,
-  TableText,
-  sortable,
-  SortByDirection,
-  headerCol,
-  TableVariant,
-  expandable,
-  compoundExpand,
-  cellWidth,
-  textCenter,
-  wrappable,
-  truncate,
-  nowrap,
-  breakWord,
-  fitContent,
-  classNames,
-  Visibility,
-  getErrorTextByValidator,
-  cancelCellEdits,
-  validateCellEdits,
-  applyCellEdits,
-  EditableTextCell,
-  EditableSelectInputCell
-} from '@patternfly/react-table';
-
-import { EmptyStateIcon } from '@patternfly/react-core';
-
 import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
 import CodeBranchIcon from '@patternfly/react-icons/dist/js/icons/code-branch-icon';
 import CodeIcon from '@patternfly/react-icons/dist/js/icons/code-icon';
 import CubeIcon from '@patternfly/react-icons/dist/js/icons/cube-icon';
 
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Table/table';
 
 import DemoSortableTable from './DemoSortableTable';
 
-## Examples
+## Table examples
 
 ### Basic
+
+The `Table` component is a configuration based component that takes a less declarative and more implicit approach about laying out the table structure, such as the rows and cells within it. For a more explicit and declarative approach, you can use the `TableComposable` component.
+
 ```js
 import React from 'react';
 
-import {
-  Table,
-  TableHeader,
-  TableBody,
-  textCenter,
-} from '@patternfly/react-table';
-import styles from '@patternfly/react-styles/css/components/Table/table';
+import { Table, TableHeader, TableBody } from '@patternfly/react-table';
 
 class SimpleTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      columns: ['Repositories', 'Branches', 'Pull requests', 'Workspaces', 'Last commit'],
+      rows: [
```

Comment:
Yeah, like this-- this is certainly outside the scope of this PR, but IMO these shouldn't be in state at all, they can just be constants and in the other examples that mutate them for selection/expandable, etc, that state could be separated and correlated with the row at render time.

Reply:
yes, would be good to update these old examples for sure! maybe outside this PR